test/simulator/items/whiteherb.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('White Herb', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it('it should use white herb after both intimidate', function () {
```

Comment:
You can leave off the extra “it” here; that’s what the function name is for.

Reply:
Thanks !! Removed